docker/nextjs/Dockerfile

```diff
@@ -1,5 +1,7 @@
 FROM node:16.14.0-alpine
 
+ENV NEXT_TELEMETRY_DISABLED=1
```

Comment:
Is there a specific technical reason for disabling telemetry, or are we just being private?

Reply:
Figure we don't need overhead of sending tracking data in background in production.